windows/CodePush.Shared/CodePushConstants.cs

```diff
@@ -27,7 +27,7 @@ internal class CodePushConstants
         internal const string UnzippedFolderName = "unzipped";
 #if WINDOWS_UWP
         internal const string AssetsBundlePrefix = "ms-appx:///ReactAssets/";
-        internal const string FileBundlePrefix = "ms-appdata:///local";
+        internal const string FileBundlePrefix = "ms-appdata:///local/";
```

Comment:
@abodalevsky, do I get it right if `FileBundlePrefix` const is used in `GetJavaScriptBundleFileAsync` method only, so there are no possible issues with this change?